data/base-spells.yaml

```diff
@@ -1559,6 +1647,11 @@ spell_data:
     abbrev: TR
     mana: 5
     cyclic: true
+  Turmar Illumination:
+    abbrev: TU IL
```

Comment:
https://elanthipedia.play.net/Turmar_Illumination says the abbrev here is `TURI`. Could you double check this in game?

Reply:
Validated that the abbrev is TU IL

Reply:
Technically both work:

>prep TURI 5
You raise your hands skyward, chanting the equation of the Turmar Illumination spell.

Reply:
Thanks for that. We should probably use the official abbreviation